### DISCUSSION TITLE
我想给项目加一个代理服务器 因为他现在报跨宇错误 ！！！ 很急

### DISCUSSION THREAD

**Discussion by 2010363329** [vercel/next.js#59143] [Help]
### Summary

我找遍官网 好像也没看见这个配置 但是vite好像有这个功能 next.js是否有通过配置加一个代理服务器的功能 或者说 还有什么更好的方案推荐（除了nginx转发 因为我感觉本地搭nginx会更加的麻烦 ） 求解答 很急！！！

**Comment by 2010363329** [comment]
有什么办法能解决代理的问题呢 help！！！

**Reply by catgrandfa** [reply]
https://nextjs.org/docs/app/api-reference/next-config-js/rewrites
或者使用api路由，代理到服务端
